### PR TITLE
Add vuejs paginator lib

### PR DESCRIPTION
**Pull request by hootlex**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/hootlex/vuejs-paginator
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars:0
   - Count of watchers:0
   - Count of forks:0
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/vuejs-paginator/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "dependencies": {
+    "vue": "^1.0.8",
+    "vue-resource": "^0.7.2"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/hootlex/vuejs-paginator.git",
+    "basePath": "dist",
+    "files": [
+      "**/*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/vuejs-paginator/package.json]: What about `"**/*"`? Don't you want to host map file? Or maybe some other files in future.

**Comment by hootlex** [comment]
You are right, i updated it.

**Comment by PeterDaveHello** [comment]
but didn't see the map file in this PR, should also add it.

**Comment by hootlex** [comment]
done!

**Comment by PeterDaveHello** [comment]
@hootlex last thing, just noticed that you don't have valid git tags for each version, but you have npm package for this lib, we aslo support npm auto-update, maybe you can change to use it instead of git auto-update?

**Comment by hootlex** [comment]
why do you say that I don't have valid git tags?

**Comment by PeterDaveHello** [comment]
https://github.com/hootlex/vuejs-paginator/tags

**Comment by hootlex** [comment]
I see. If I make the tags same with npm will it be ok?

**Comment by hootlex** [comment]
I create the one which exists on npm. https://github.com/hootlex/vuejs-paginator/releases/tag/V1.0.11

**Comment by PeterDaveHello** [comment]
Don't you want to also add the old versions?

**Comment by hootlex** [comment]
No, this is the only working one.

**Pull request merged**